### PR TITLE
Cabana: add window icon

### PR DESCRIPTION
**Pull request by sshane**
@deanlee is there a better way to do this?

Screenshot from 2023-01-26 20-42-58

### CODE REVIEW

**Comment by deanlee** [comment]
it's good.:D

**Comment by adeebshihadeh** [comment]
Lol, let's at least make this a bit nicer. Maybe just bigger comma and make it a circle?

File: tools/cabana/cabana.cc [hunk 1]
```diff
@@ -13,6 +13,7 @@ int main(int argc, char *argv[]) {
   initApp(argc, argv);
   QApplication app(argc, argv);
   app.setApplicationDisplayName("Cabana");
+  app.setWindowIcon(QIcon("./images/cabana.png"));
```
Comment (pd0wm) [inline tools/cabana/cabana.cc:16]: You want to use qrc to load the file, otherwise it won't work if your current working directory is wrong.

**Comment by sshane** [comment]
@deanlee do you want to make a PR to add support for a tools/cabana qrc file? That's the only thing left for this :)

**Comment by deanlee** [comment]
@sshane  added qrc resource file in https://github.com/commaai/openpilot/pull/27386

**Comment by sshane** [comment]
@deanlee away from my desktop and cabana doesn't yet work on Win11, can you test this out?

**Pull request merged**